examples/example-sandstream.html

```diff
@@ -0,0 +1,83 @@
+<!doctype html>
+<html dir="ltr" lang="en-US">
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta http-equiv="content-type" content="text/html; charset=utf-8">
+    <link href="style.css" rel="stylesheet">
+    <script src="../src/bundle.js" data-cfasync="false"></script>
+  </head>
+<body>
+  <div id="btn-revokeChoice" style="color: white; padding: 32px">Change my consent</div>
+  
+  <script>
+    const CC = window.CookieConsent;
+      const cc = new CC({
+        // just some default styling
+        "palette": {
+          "popup": {
+            "background": "black",
+            "text": "white"
+          },
+          "button": {
+            "background": "#f5d948"
+          }
+        },
+        type: 'opt-in',
+        consentSettingsElemId: 'btn-revokeChoice',
+        layout: 'sanddev',
+        showCategories: {
+          [CC.UNCATEGORIZED]: false,
+          [CC.ESSENTIAL]: true,
+          [CC.PERSONALIZATION]: false,
+          [CC.ANALYTICS]: true,
+          [CC.MARKETING]: false
+        },
+        content: {
+          privacyPolicyLink: '/privacy-policy',
+          cookiePolicyLink: '/cookie-policy'
+        }
+      });
+
+      cc.on('initialized', function () {
+        const statusesMap = cc.consents();
+
+        if (statusesMap[CC.ANALYTICS] === CC.ALLOW) {
+          initializeGTM();
+        }
+
+        console.info('initialized', statusesMap);
```

Comment:
I am not fan of `console.info` - just `console.log` (stdout) and `console.error` (stderr) seems to be reasonable.